tests/vcr_python_wrapper.py

```diff
@@ -47,6 +47,19 @@ def domain_query_matcher(r1, r2):
     return r1.query == r2.query
 
 
+def katello_manifest_body_matcher(r1, r2):
+    if r1.headers.get('content-type').startswith('multipart/form-data') and r2.headers.get('content-type').startswith('multipart/form-data'):
+        if r1.path.endswith('/subscriptions/upload') and r2.path.endswith('/subscriptions/upload'):
+            r1_copy = vcr.request.Request(r1.method, r1.uri, r1.body, r1.headers)
+            r2_copy = vcr.request.Request(r2.method, r2.uri, r2.body, r2.headers)
+            # the body is a huge binary blob, which seems to differ on every run, so we just ignore it
+            body1 = body2 = {}
+            r1_copy.body = json.dumps(body1)
+            r2_copy.body = json.dumps(body2)
+            return body_json_l2_matcher(r1_copy, r2_copy)
```

Comment:
I guess we don't need that line. We can just fall through.

Reply:
... if we reused the names r1 and r2.

Reply:
Yes, but for some reason we've been keeping this as copies. And I didn't want to try to understand why ;)

Reply:
If you change the original requests, that is, what is persisted in the cassettes. (And maybe even sent to the server.)
But instead of naming it r1_copy we could just reuse the local name r1. We would still only modify the copy.